src/components/button.tsx

```diff
@@ -46,19 +46,19 @@ export const Button: React.StatelessComponent<ButtonProps> =
             }
         }
 
-        if (type === 'link' && isLinkExternal) {
+        if (type === 'link' && isLinkInternal) {
```

Comment:
Not a big fan of this if/else if/else thing. Maybe use a switch, or better even map the types to a render function.